src/interaction-handlers/voteReminderButton.ts

```diff
@@ -8,23 +9,17 @@ import { ButtonInteraction, TimestampStyles, time } from 'discord.js';
 @ApplyOptions<InteractionHandler.Options>({ interactionHandlerType: InteractionHandlerTypes.Button })
 export class VoteReminderButton extends InteractionHandler {
 	public override parse(interaction: ButtonInteraction) {
-		if (interaction.customId !== ButtonID.voteReminder) return this.none();
+		if (interaction.customId !== 'vote-reminder-button') return this.none();
```

Comment:
Having the Button IDs that we are using in a seperate place makes sense right? Why the change? In my view this would possibly lead to duplicate code and make it harder to maintain!

Reply:
https://github.com/BirthdayyBot/BirthdayyBot/pull/568/commits/2469eacd870bbc8a96ce696f91cc8977a71d9021